articles/nodejs-frontend-backend/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/nodejs-frontend-backend/
+title: Node.js - Frontend or Backend?
+description: Node.js is a runtime environment for executing server-side code with higher efficiency and it presents a larger bandwidth to handle large code payloads.
+author: lalithnarayan-c
+date: 2020-08-20T00:00:00-15:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/nodejs-frontend-backend/hero.jpg
+    alt: front end and backend with Node.js
+---
+Let us start by understanding what Node.js is. Node.js is a runtime environment for executing server-side code with higher efficiency and it presents a larger bandwidth to handle large code payloads. The objective of this article is as follows-
+<!--more-->
+
+1. Understand and clarify any doubt of whether Node.js is frontend or backend application
+2. List out various applications in both frontend and backend
+3. Understand how Node.js complements other technologies in the frontend and backend
+
+The choice of the language that Ryan Dahl, the creator of Node.js, opted for is the most popular programming language, JavaScript.
```

Comment:
would be helpful to add a source - anything that is not common language lets add a source @lalith1403